app/frontend/src/features/userPage/ResponsiveAvatar.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { User } from "../../pb/api_pb";
+import { makeStyles } from "@material-ui/core";
+import PageTitle from "../../components/PageTitle";
+import Avatar from "../../components/Avatar";
+
+const useStyles = makeStyles({
+  root: {
+    position: "relative",
+    width: "100%",
+    paddingTop: "100%",
+  },
+  avatar: {
+    position: "absolute",
+    left: 0,
+    right: 0,
+    top: 0,
+    bottom: 0,
+    width: "100%",
+    height: "100%",
+  },
+});
+
+export default function ReponsiveAvatar({ user }: { user: User.AsObject }) {
+  const classes = useStyles();
+  return (
+    <>
+      <div className={classes.root}>
+        <Avatar user={user} className={classes.avatar} />
+      </div>
+    </>
```

Comment:
Is this an ok way to get a responsive avatar? `height: auto` didn't work.

Reply:
Assumptions: All Avatars must be responsive. There are only a few different sizes: avatars in list items, as profile photo, others?
To make the responsivity less complex we can define the default size in the avatar component relative to the font-size. For instance: font-size: 3em
The font-size will be relative to the font-size of the body element: font-size: 1rem
That font-size is relative to the width of the viewport: font-size: 3vw

Reply:
See also the PR about the Avatar story I have just posted.